Clockwise/VirtualClock.cs

```diff
@@ -94,12 +92,23 @@ private VirtualClock(DateTimeOffset? now = null, [CallerMemberName] string creat
                 scheduledTime = after.Value;
             }
 
-            while (schedule.ContainsKey(scheduledTime))
```

Comment:
This while loop is an O(log2(m) * n) operation where 

m = number of elements in schedule
n = number of times that a conflict occurs.

The current replacement is O(m), but I have plans to upgrade it to O(log2(m) + n)